### PR TITLE
Add infoscreen to the fullstack, fix stripe_pk to docker-compose

### PR DESCRIPTION
**Pull request by HenrikEngen**
Minor changes to docker-compose.yml and nginx.conf

* Added the infoscreen app to docker-compose.yml.
* Added the infoscreen app as a server in nginx.conf.
* ~~Added STRIPE_PK: " " to fix an issue where STRIPE didnt work if this was not set.~~
* ~~Exposed port 5432 to the db container in docker-compose to allow connections and modifications through ex. pgAdmin.~~

### CODE REVIEW

File: phoenix-development/docker-compose.yml [hunk 1]
```diff
@@ -26,6 +26,7 @@ services:
       HOST: "http://delta.dev.phoenixlan.no:3000"
       REACT_APP_OAUTH_CLIENT_ID: "phoenix-delta-dev"
       REACT_APP_MAIN_SITE: "http://dev.phoenixlan.no:3000"
+      STRIPE_PK: " "
```
Comment (petterroea) [inline phoenix-development/docker-compose.yml:29]: Wrong spot to place a STRIPE_PK. Look at the rest of the file. We use environment-files for secrets.

**Review by petterroea — CHANGES_REQUESTED** [review]

File: phoenix-development/docker-compose.yml [hunk 2]
```diff
@@ -113,9 +114,28 @@ services:
...
+    volumes:
+      - ../../phoenixinfo-v1:/srv/phoenix
+      - ../../phoenixjs:/srv/phoenixjs
+    logging:
+      options:
+        max-size: "1g"
   db:
     image: postgres:alpine
     env_file: .db_env
+    ports:
```
[the remark this answers is not in the document]
Comment (petterroea) [reply phoenix-development/docker-compose.yml:137]: Alternatively we could add adminer directly to the docker-compose file as a GUI alternative. Let's not open up a bunch of ports on our developers PCs without any need. 

Using pgAdmin from outside docker to access a database that is _inside_ docker-compose is pretty odd, isn't it?

[the remark this answers is not in the document]
Comment (HenrikEngen) [reply phoenix-development/docker-compose.yml:137]: > No plans of exposing postgres. Use the command line (`docker-compose exec db psql --username=phoenix`)
I agree, my bad. We dont want to expose the database, especially not in prod. This was meant to stay local for development only to make it easier to modify the database with pgAdmin.

[the remark this answers is not in the document]
Comment (petterroea) [reply phoenix-development/docker-compose.yml:137]: No worries, remember that this file is only for local development. Still, remember that this does open a port on peoples laptop. Even at our LAN party, some kids do port scan. I have been asked "why do you have this running on your laptop" by a kid before. Not fun :D

[the remark this answers is not in the document]
Comment (petterroea) [reply phoenix-development/docker-compose.yml:137]: I much prefer that people get comfortable with using `psql`, or that we bundle a lightweight db admin tool

File: phoenix-development/docker-compose.yml (hunk 1 above)
Comment (HenrikEngen) [reply phoenix-development/docker-compose.yml]: Got it. Removed from the code and placed in a local environment file.

**Pull request merged**